source/codegen/templates/service_helpers.mako

```diff
@@ -158,13 +158,21 @@ ${initialize_standard_input_param(function_name, parameter)}\
   request_snippet = f'request->{field_name}()'
   c_type = parameter['type']
   c_type_pointer = c_type.replace('[]','*')
+  c_type_underlying_type = common_helpers.get_underlying_type_name(c_type)
 %>\
 % if c_type == 'ViConstString':
       ${c_type} ${parameter_name} = ${request_snippet}.c_str();\
 % elif c_type == 'ViString' or c_type == 'ViRsrc':
       ${c_type} ${parameter_name} = (${c_type})${request_snippet}.c_str();\
 % elif c_type == 'ViInt8[]' or c_type == 'ViChar[]':
       ${c_type_pointer} ${parameter_name} = (${c_type[:-2]}*)${request_snippet}.c_str();\
+% elif c_type == 'ViBoolean[]':
```

Comment:
Since right now there are no functions with ViBoolean[] input parameter, I changed in_compliance of FetchMultiple to "in" and generated code for validating. Here's what generated code for input parameter would look like.

![image](https://user-images.githubusercontent.com/7554740/115590446-97586180-a2ee-11eb-8ebc-c2e12e977f9a.png)